SpringProject/src/main/java/com/gradproject2019/users/service/UserServiceImpl.java

```diff
@@ -13,15 +15,26 @@
 
     private UserRepository userRepository;
 
-    public UserServiceImpl(UserRepository userRepository) {
+    @Autowired
```

Comment:
You can remove @Autowired from here because your services are injected in constructor. 
Also I would recommend to put your services as final, it's a good practice to make them final when you inject them via constructor.